config/environments/development.rb

```diff
@@ -34,9 +34,9 @@
   end
 
   # Store uploaded files on the local file system (see config/storage.yml for options).
-  # config.active_storage.service = :local
+  config.active_storage.service = :local
 
-  config.active_storage.service = :amazon
+  # config.active_storage.service = :amazon
```

Comment:
isso não é necessario. 
se vc esta configurando o `config/environments/development.rb`, logo o `active_storage.service` deve ser local. o nome ja diz, `environments/development`. não precisa ficar trocando ou comentando esse valor.
ele será `config.active_storage.service = :amazon` quando o arquivo for `config/environments/production.rb` e ou `config/environments/staging.rb`. não tem q ficar comentando o que não irá usar.

Reply:
Eu fico comentando o da amazon para nao correr o risco de gastar na aws.
Commitei por engano esse arquivo

Reply:
é so comentar o valor da chave no `.env` da sua maquina que nada será utilizado.

Reply:
[1288cf4](https://github.com/Lucas-SilvaGit/economease/pull/28/commits/1288cf499727ba90aa8e04381d90d51741e17529)